pkg/tfvars/aws/aws.go

```diff
@@ -4,38 +4,51 @@ package aws
 import (
 	"encoding/json"
 	"fmt"
+	"sort"
 
 	"github.com/openshift/installer/pkg/types/aws/defaults"
 	"github.com/pkg/errors"
 	"sigs.k8s.io/cluster-api-provider-aws/pkg/apis/awsproviderconfig/v1beta1"
 )
 
 type config struct {
-	AMI                   string            `json:"aws_ami"`
-	ExtraTags             map[string]string `json:"aws_extra_tags,omitempty"`
-	BootstrapInstanceType string            `json:"aws_bootstrap_instance_type,omitempty"`
-	MasterInstanceType    string            `json:"aws_master_instance_type,omitempty"`
-	AvailabilityZones     []string          `json:"aws_master_availability_zones"`
-	IOPS                  int64             `json:"aws_master_root_volume_iops"`
-	Size                  int64             `json:"aws_master_root_volume_size,omitempty"`
-	Type                  string            `json:"aws_master_root_volume_type,omitempty"`
-	Region                string            `json:"aws_region,omitempty"`
+	AMI                     string            `json:"aws_ami"`
+	ExtraTags               map[string]string `json:"aws_extra_tags,omitempty"`
+	BootstrapInstanceType   string            `json:"aws_bootstrap_instance_type,omitempty"`
+	MasterInstanceType      string            `json:"aws_master_instance_type,omitempty"`
+	MasterAvailabilityZones []string          `json:"aws_master_availability_zones"`
```

Comment:
why do we care about master and worker azs separately in terraform. just give the list of azs...

Reply:
> why do we care about master and worker azs separately in terraform...

Because we're actually [*creating* masters][1].  If we flatten here, you might get masters in the wrong spots.

[1]: https://github.com/openshift/installer/blob/009963e27328d688ebf3f1d475973056ef40cc52/data/data/aws/main.tf#L35